src/Service/AddFormService.php

```diff
@@ -108,4 +108,20 @@ public static function isEmptyDataJump(array $jumplog, array $errorMessages): ar
         }
         return $errorMessages;
     }
+
+    public static function validateUrl(array $jumplog, array $errorMessages): array
+    {
+        if ($jumplog['video'] != filter_var($jumplog['video'], FILTER_VALIDATE_URL)) {
+            $errorMessages[] = "merci de renseigner une url pour le champ vidéo";
+        }
+        return $errorMessages;
+    }
+
+    public static function checkDate(array $jumplog, array $errorMessages): array
+    {
+        if ($jumplog['date_of_jump'] < 2022) {
```

Comment:
No, if you need to change ths line every year it's a bad thing. Find a way to get the actual year in PHP.

Reply:
Ok, but that's a bit weird : If i want to post a jump i did in 2021 I can't BUT in three years if i want to post a jump I did in 2022, it will be OK. I'd like better : I can post a jump older than 3 years <-- it will always be the same rule.

Reply:
yes i change for date('Y')